controllers/login.js

```diff
@@ -286,14 +286,33 @@ router.get('/login/success', authHelper.authChecker, async (req, res, next) => {
 	return null;
 });
 
+// eslint-disable-next-line consistent-return
+const sessionDestroyer = (req, res, rej, next) => {
+	if (req.url === '/logout') {
+		req.session.destroy((err) => {
+			if (err) {
+				rej(`Error destroying session: ${err}`);
+			} else {
+				// clear the CSRF token to prevent re-use after logout
+				res.locals.csrfToken = null;
+			}
+			return next();
+		});
+	} else {
+		return next();
+	}
+};
```

Comment:
```suggestion
const sessionDestroyer = (req, res, rej, next) => {
	if (req.url === '/logout') {
		req.session.destroy((err) => {
			if (err) {
				rej(`Error destroying session: ${err}`);
			} else {
				// clear the CSRF token to prevent re-use after logout
				res.locals.csrfToken = null;
			}
		});
	}
	return next();
};
```